gruntfile.js

```diff
@@ -10,7 +10,14 @@ module.exports = function(grunt)
                     basePath: 'themes',
                     declaration: false
                 },
-                src: ['src/default/assets/js/src/**/*.ts'],
+                src: [
+                    'src/default/assets/js/src/lib/**/*.ts',
+                    'src/default/assets/js/src/typedoc/Application.ts',
+                    'src/default/assets/js/src/typedoc/components/**/*.ts',
+                    'src/default/assets/js/src/typedoc/services/**/*.ts',
+                    'src/default/assets/js/src/typedoc/utils/**/*.ts',
```

Comment:
> all client js is currently not working without changing concat order on my system. does it work on yours?

I haven't tested it, sorry. I'm just helping maintain the issues and PRs.

> so an alternative globbing might be 

Sorry, yes, this is what I was suggesting. I wasn't suggesting the same thing it was before. I guess I hoped the location of my comment made it clearer, but I suppose not 😄